package.json

```diff
@@ -27,36 +28,38 @@
     }
   ],
   "dependencies": {
-    "ember-cli-babel": "^6.10.0",
-    "ember-cli-htmlbars": "^2.0.1",
+    "ember-cli-babel": "^7.1.2",
```

Comment:
this changes the minimum supported Ember CLI version to 2.13. this should be considered a breaking change.

Reply:
Thanks for pointing this out Tobias. Should we release as a 2.x beta then?

Reply:
probably, yeah